spec/controllers/budgets_controller_spec.rb

```diff
@@ -80,8 +104,8 @@
 
       it "renders edit" do
         user = create(:user)
-        budget = create(:budget, user: user)
         allow(controller).to receive(:current_user).and_return(user)
+        budget = create(:budget, user: user)
```

Comment:
I usually create the objects and then stub the controller but really up to you

Reply:
No, I'd rather do it the correct, more sensible way, thanks Elle. Done in PR ad-budget-frequency-amounts sha: 292cc37